test/Generators/Microsoft.Gen.AutoClient/Generated/QueryTests.cs

```diff
@@ -80,6 +80,37 @@ public async Task QueryIsEscaped()
         Assert.Equal("Success!", response);
     }
 
+    [Theory]
+    [InlineData("?")]
+    [InlineData("=")]
+    [InlineData("&")]
+    [InlineData("%")]
+    [InlineData("+")]
+    [InlineData("#")]
+    [InlineData(" ")]
+    [InlineData("/")] // / isn't required to be escaped in query string values but there is no harm in doing so
```

Comment:
is there any value in a test that puts every other ASCII char in and checks none are escaped?

Reply:
&#x55;&#x6E;&#x6C;&#x69;&#x6B;&#x65;&#x6C;&#x79;&#x2E;&#x20;&#x49;&#x74;&#x27;&#x73;&#x20;&#x70;&#x65;&#x72;&#x66;&#x65;&#x63;&#x74;&#x6C;&#x79;&#x20;&#x76;&#x61;&#x6C;&#x69;&#x64;&#x20;&#x74;&#x6F;&#x20;&#x65;&#x73;&#x63;&#x61;&#x70;&#x65;&#x20;&#x65;&#x76;&#x65;&#x72;&#x79;&#x20;&#x63;&#x68;&#x61;&#x72;&#x61;&#x63;&#x74;&#x65;&#x72;&#x3B;&#x20;&#x74;&#x68;&#x65;&#x20;&#x6F;&#x74;&#x68;&#x65;&#x72;&#x20;&#x65;&#x6E;&#x64;&#x70;&#x6F;&#x69;&#x6E;&#x74;&#x20;&#x77;&#x69;&#x6C;&#x6C;&#x20;&#x64;&#x6F;&#x20;&#x74;&#x68;&#x65;&#x20;&#x72;&#x69;&#x67;&#x68;&#x74;&#x20;&#x74;&#x68;&#x69;&#x6E;&#x67;&#x2E;&#x20;&#x49;&#x6E;&#x20;&#x66;&#x61;&#x63;&#x74;&#x2C;&#x20;&#x65;&#x76;&#x65;&#x72;&#x79;&#x20;&#x73;&#x69;&#x6E;&#x67;&#x6C;&#x65;&#x20;&#x63;&#x68;&#x61;&#x72;&#x61;&#x63;&#x74;&#x65;&#x72;&#x20;&#x6F;&#x66;&#x20;&#x74;&#x68;&#x69;&#x73;&#x20;&#x63;&#x6F;&#x6D;&#x6D;&#x65;&#x6E;&#x74;&#x20;&#x69;&#x73;&#x20;&#x65;&#x73;&#x63;&#x61;&#x70;&#x65;&#x64;&#x20;&#x28;&#x75;&#x73;&#x65;&#x20;&#x74;&#x68;&#x65;&#x20;&#x74;&#x68;&#x72;&#x65;&#x65;&#x20;&#x64;&#x6F;&#x74;&#x73;&#x20;&#x74;&#x6F;&#x20;&#x73;&#x65;&#x65;&#x20;&#x69;&#x74;&#x21;&#x29;&#x2C;&#x20;&#x61;&#x6E;&#x64;&#x20;&#x69;&#x74;&#x20;&#x73;&#x68;&#x6F;&#x75;&#x6C;&#x64;&#x20;&#x72;&#x65;&#x6E;&#x64;&#x65;&#x72;&#x20;&#x63;&#x6F;&#x72;&#x72;&#x65;&#x63;&#x74;&#x6C;&#x79;&#x20;&#x69;&#x6E;&#x20;&#x79;&#x6F;&#x75;&#x72;&#x20;&#x62;&#x72;&#x6F;&#x77;&#x73;&#x65;&#x72;&#x2E;